### PR TITLE
Add aria-selected attribute to record tabs for accessibility.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by demiankatz** [comment]
@aidaluuppala, I haven't studied this in great depth, so it's entirely possible that you've already thought about this and your approach is the most efficient way... but just to brainstorm a possible alternative: there is already a lot of logic in the code to add and remove the "active" class from tabs. Might it be more streamlined if you abstracted this into "activateTab / deactivateTab" functions and then incorporated the aria-selected logic into those functions? If I'm understanding correctly, the process of activating and deactivating tabs is directly linked to the need for aria-selected, so I'm wondering if it would be more clear to treat those as two steps of the same function rather than handling the two operations independently of one another.

What do you think? (And my apologies if I'm thinking about this incorrectly!) :-)

**Comment by ghost** [comment]
@demiankatz thank you and good point, I'll try and incorporate this with the active class logic.

**Comment by ghost** [comment]
@demiankatz It seems that the logic of active class is in the bootstrap library, but now there is an event listener that is listening to tab shown event, so it should be a bit more streamlined.

**Comment by demiankatz** [comment]
Thanks, @aidaluuppala, this is much simpler and easier to understand! :-) I'm asking @crhallberg to give it a review as well since he's more familiar with our UI code and might have some suggestions that I overlooked. Once he approves, I think it's ready to merge. Since I'll be away for the next two weeks, he's welcome to hit the merge button in my absence if he's happy with everything.

**Comment by crhallberg** [comment]
Thank you for your patience!

File: themes/bootstrap3/js/record.js
Comment (crhallberg) [inline themes/bootstrap3/js/record.js:321]: This code is great! A comment or two might make it more readable.

**Review by crhallberg — APPROVED** [review]

File: themes/bootstrap3/js/record.js
Comment (demiankatz) [reply themes/bootstrap3/js/record.js:321]: Thanks for the review, @crhallberg. I'll wait to see if @aidaluuppala decides to add comments before merging this.

**Comment by demiankatz** [comment]
Thanks, @aidaluuppala!

**Pull request merged**
